app/views/govuk_publishing_components/components/_taxonomy_navigation.html.erb

```diff
@@ -0,0 +1,115 @@
+<% if local_assigns[:items] && !items.blank? %>
+  <aside class='gem-c-taxonomy-navigation' data-module='track-click' role='complementary'>
+    <h2 class='gem-c-taxonomy-navigation__heading'><%= t("components.taxonomy_navigation.related_content") %></h2>
+    <% items.each_with_index do |item, item_index| %>
+      <div data-track-count="sidebarTaxonSection">
+        <h3 class='gem-c-taxonomy-navigation__heading'>
+          <%=
+            link_to_if(
+              item[:url],
+              item[:title],
+              item[:url],
+              class: 'gem-c-taxonomy-navigation__link',
+              data: {
+                track_category: 'relatedLinkClicked',
+                track_action: "#{item_index + 1}",
+                track_label: item[:url],
+                track_options: {
+                  dimension28: items.length.to_s,
+                  dimension29: item[:title],
+                },
+              },
+            )
+          %>
+        </h3>
+
+        <p class='gem-c-taxonomy-navigation__taxon-description'>
+          <%= item[:description] %>
+        </p>
+
+        <% item[:related_content] ||= [] %>
+        <% if item[:related_content].any? %>
+          <nav class='gem-c-taxonomy-navigation__navigation' role='navigation'>
```

Comment:
Just noticed this: you need to add `data-module="toggle"` to the nav, as otherwise the toggle button will append '#' to the end of the URL instead of actually toggling the content. A toggle example should probably be added to the component guide so this can be checked when someone makes changes to the component in the future.

Reply:
Good catch! I'll fix and add an example to the guide. :+1:

Reply:
@vanitabarrett added and example https://github.com/alphagov/govuk_publishing_components/pull/253/commits/9776414b1cf26538219685a40bd33272d437d40c